salt/apps/odlvideo/configure.sls

```diff
@@ -0,0 +1,14 @@
+include:
+    - uwsgi.service
+
+create_env_file_for_odlvideo:
+  file.managed:
+    - name: /opt/{{ salt.pillar.get('django:app_name') }}/.env
+    - contents: |
+        {%- for var, val in salt.pillar.get('django:environment').items() %}
+        {{ var }}={{ val }}
+        {%- endfor %}
+    - require:
+        - deploy_application_source_to_destination
```

Comment:
This require statement will probably fail when executed in isolation as `django.config` but is probably not needed now that it is getting run at a different step in the execution.